diff --git a/shared/common/src/28_numeric_noabirule/CMakeLists.txt b/shared/common/src/28_numeric_noabirule/CMakeLists.txt
index 6860fcd304..53168312c5 100644
--- a/shared/common/src/28_numeric_noabirule/CMakeLists.txt
+++ b/shared/common/src/28_numeric_noabirule/CMakeLists.txt
@@ -5,15 +5,12 @@ add_library(28_numeric_noabirule STATIC
   m_abi_linalg.F90
   m_array.F90
   m_bessel.F90
-  m_bessel2.F90
-  m_brentq.F90
   m_bspline.F90
   m_cplxtools.F90
   m_elpa.F90
   m_gaussian_quadrature.F90
   m_hide_blas.F90
   m_hide_lapack.F90
-  m_hybrd.F90
   m_lebedev.F90
   m_levenberg_marquardt.F90
   m_lib_four.F90
@@ -23,7 +20,6 @@ add_library(28_numeric_noabirule STATIC
   m_random_xoroshiro128plus.F90
   m_random_zbq.F90
   m_slk.F90
-  m_slsqp.F90
   m_sort.F90
   m_splines.F90
   )
diff --git a/src/65_paw/CMakeLists.txt b/src/65_paw/CMakeLists.txt
index b05c4e294f..918ec484f7 100644
--- a/src/65_paw/CMakeLists.txt
+++ b/src/65_paw/CMakeLists.txt
@@ -10,7 +10,6 @@ add_library(65_paw STATIC
   m_paw_dfptnl.F90
   m_paw_dmft.F90
   m_paw_efield.F90
-  m_paw_exactDC.F90
   m_paw_hr.F90
   m_paw_init.F90
   m_paw_lmn.F90
@@ -26,7 +25,6 @@ add_library(65_paw STATIC
   m_paw_sym.F90
   m_paw_tools.F90
   m_paw_uj.F90
-  m_paw_yukawa.F90
   m_pawfgr.F90
   m_pawpwij.F90
   )
