--- a/src/vout.f90	2024-06-01 11:30:38.000000000 +0300
+++ b/src/vout.f90	2024-06-01 11:33:15.000000000 +0300
@@ -130,8 +130,6 @@
    call excpzca(rhot,vxc,exca,mmax)
  else if(iexc .eq. 4) then
    call excggc(rhot,vxc,exca,rr,mmax)
- else if (iexc < 0) then
-   call exc_libxc(iexc,al,rhot,vxc,exca,rr,mmax)
  else
    write(6,'(/a,i4)') 'vout: ERROR bad input iexc =',iexc
    stop
@@ -162,8 +160,6 @@
      call excpzca(rhoc,vxcd,excca,mmax)
    else if(iexc .eq. 4) then
      call excggc(rhoc,vxcd,excca,rr,mmax)
-   else if (iexc < 0) then
-     call exc_libxc(iexc,al,rhoc,vxcd,excca,rr,mmax)
    else
      write(6,'(/a,i4)') 'vout: ERROR bad input iexc =',iexc
      stop
