+++ a/lib/migrations/20200321153000-fix-account-deletion.js	2021-05-09 10:12:23.201236644 +0300
--- b/lib/migrations/20200321153000-fix-account-deletion.js	2021-05-09 10:12:18.141247959 +0300
***************
*** 4,12 ****
  module.exports = {
    up: function (queryInterface, Sequelize) {
      const cleanup = spawnSync(process.argv[0], ['./bin/cleanup'], { cwd: path.resolve(__dirname, '../../') })
-     if (cleanup.status !== 0) {
-       throw new Error('Unable to cleanup')
-     }
      return queryInterface.addConstraint('Notes', ['ownerId'], {
        type: 'foreign key',
        name: 'Notes_owner_fkey',
--- 4,9 ----
